modules/kernel/src/org/apache/axis2/context/ConfigurationContext.java

```diff
@@ -890,6 +891,22 @@ public void setServicePath(String servicePath) {
         this.servicePath = servicePath;
     }
 
+    /**
+     * Set the ServicePathValidation parameter
+     * @param servicePathValidation service path validation enabled or not
+     */
+    public void setServicePathValidation(boolean servicePathValidation) {
+        this.servicePathValidation = servicePathValidation;
+    }
+
+    /**
+     * Get the ServicePathValidation parameter
+     * @return  service path validation enabled or not
+     */
+    public boolean getServicePathValidation() {
```

Comment:
isn't the method name should be isStrictServicePathValidationEnabled() ?

Reply:
+1 fixed